integreat_cms/cms/views/contacts/contact_form_ajax_view.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from django.http import JsonResponse
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.views.generic import TemplateView
+
+from ...decorators import permission_required
+from ...forms import ContactForm
+from ...models import Contact
+from .contact_context_mixin import ContactContextMixin
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from django.http import HttpRequest, HttpResponse
+
+
+@method_decorator(permission_required("cms.view_contact"), name="dispatch")
+@method_decorator(permission_required("cms.change_contact"), name="post")
+class ContactFormAjaxView(TemplateView, ContactContextMixin):
+    """
+    View for the ajax contact widget
+    """
+
+    #: Template for ajax POI widget
+    template = "ajax_contact_form/_contact_form_widget.html"
+
+    def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        r"""Render a contact form widget template
+
+        :param request: The current request
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        :return: The html template of a POI form
+        """
+        contact_form = ContactForm(
+            additional_instance_attributes={"region": request.region}
+        )
+
+        return render(
+            request,
+            "ajax_contact_form/_contact_form_widget.html",
+            {
+                **self.get_context_data(**kwargs),
+                "contact_form": contact_form,
+                "poi_id": kwargs.get(
+                    "poi_id",
+                ),
+            },
+        )
+
+    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        r"""Add a new POI to the database
+
+        :param request: The current request
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        :raises ~django.http.Http404: If no language for the given language slug was found
+
+        :return: A status message, either a success or an error message
+        """
+
+        contact_instance = Contact.objects.filter(id=None).first()
```

Comment:
This will always be `None`, right? There cannot be an object without an ID.

(I assume this was copied from the `POIAjaxFormView`? I think it is an error there, too)

Reply:
You're right 👀 😁 Fixed. The one in `POIAjaxForm` will be fixed as #3359 (as not in the scope of this PR)